internal/services/containers/kubernetes_cluster_addons_resource_test.go

```diff
@@ -338,6 +338,7 @@ resource "azurerm_kubernetes_cluster" "test" {
   location            = azurerm_resource_group.test.location
   resource_group_name = azurerm_resource_group.test.name
   dns_prefix          = "acctestaks%d"
+  node_resource_group = "${azurerm_resource_group.test.name}-nodes"
```

Comment:
Please remove all of these, we run the acceptance tests on `westeurope` and I think france is the alternative location. Capacity issues are usually temporary anyway.

Reply:
Isn't it achievable that acceptance tests works in every region?

Reply:
Yes, but `westeurope` is where we have to run the acceptance tests and the tests should not include `Optional` properties that are not required for testing the related functionality. The limitation is coming from the AKS API so I would open an issue there to get that resolved.

Reply:
If could cut 3 chars from `acctestRG-aks-240531233634701724`, it would also resolve the issue.

I will leave the change, until other issues are resolve, I will remove the lines, but I also do not want to re-add them all the times for local testing.